gc.c

```diff
@@ -1537,30 +1537,29 @@ os_obj_of(VALUE of)
  *  Ruby process. If <i>module</i> is specified, calls the block
  *  for only those classes or modules that match (or are a subclass of)
  *  <i>module</i>. Returns the number of objects found. Immediate
- *  objects (<code>Fixnum</code>s, <code>Symbol</code>s
+ *  objects (<code>Fixnum</code>s, static <code>Symbol</code>s
```

Comment:
To imply this list is not exhaustive:

```suggestion
 *  objects (such as <code>Fixnum</code>s, static <code>Symbol</code>s
```

We could also mention `most <code>Float</code>s (on 64-bit platforms)`. However, if we are going to discuss and/or give examples of floats, we should probably discuss the non-immediate cases (floats greater than `2**256` or less than `-(2**256)`, infinity, NaN, 32-bit platforms). If we don't want to discuss that, I would remove floats from the examples.

Considering that if we use `ObjectSpace.each_object(Numeric)` as an example, it's going to have include floats, I think we should either pick a different code example, or discuss these float cases.